src/components/calcite-flow-item/calcite-flow-item.tsx

```diff
@@ -29,6 +29,11 @@ export class CalciteFlowItem {
   //
   // --------------------------------------------------------------------------
 
+  /**
+   * Optionally perform a function to run before going back.
```

Comment:
Small suggestion: `When provided, this method will be called before it is removed from the parent flow.` or something like that. cc @kat10140